HULKs/hulk
# Twix CliArguments

## Introduced Changes

Adds two cli arguments to twix I was really missing
 - nao address: can just connect to a robot using the syntax known from pepsi, eg `twix 34w`
 - clear: bool flag that clears the panel setup, useful when your twix persisted state has panels that don't exist on main

Fixes #

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test

* run `twix <NAO_NUMBER>`, it should automatically have the correct nao address
* run `twix --clear`. twix opens with a blank new text panel
